charts/lfx-v2-project-service/templates/ruleset.yaml

```diff
@@ -75,9 +70,14 @@ spec:
         routes:
           - path: /projects/:id
       execute:
-        #- authenticator: authelia
+        {{- if .Values.auth0.enabled }}
+        - authenticator: auth0
+        {{- else if .Values.authelia.enabled }}
+        - authenticator: authelia
+        - contextualizer: authelia_userinfo
+        {{- else if .Values.anonymous_authenticator.enabled }}
         - authenticator: anonymous_authenticator
-        #- contextualizer: authelia_userinfo
+        {{- end }}
```

Comment:
Since all "authorized" routes will generally contain the same authenticators, and contexualizers, can we make these configurable from values similar to including additional labels or env vars?
```yaml
{{- with .Values.heimdall.executors }}
{{ toYaml . | indent 8 }}
{{- end }}
```

This way the default could be defined as:
```yaml
heimdall:
  executors:
     - authenticator: authelia
     - authenticator: anonymous_authenticator
     - contextualizer: authelia_userinfo
```

And overridden like:
```yaml
heimdall:
  executors:
    - authorizer: auth0
```